daboizzBot/main.py

```diff
@@ -0,0 +1,12 @@
+# Good practice to include this. 
+import os
+from dotenv import load_dotenv
+from daboizzBot.bot import create_bot
+
+
+if __name__ == "__main__":

```

Comment:
This checks for if this is the entry point. Always recommend something like this. 

Alternatively you could do something like the following

```Python
def main():
    load_dotenv() # Will load environment variables from a .env file
    TOKEN = os.getenv("PRIVATE_KEY")
    client = create_bot()
    #Run Bot
    client.run(TOKEN) 
```

The in the `Procfile` have the following.
```
worker: python daboizzzBot/main.py::main
```